lib/api/attributes.js

```diff
@@ -1,10 +1,12 @@
 var _ = require('lodash'),
+  static = require('../static'),
```

Comment:
It looks like we are using `$` for static in other files. I am going to switch to that for consistency for now:

https://github.com/cheeriojs/cheerio/blob/0.18.0/lib/api/manipulation.js#L3